main/SAPI.h

```diff
@@ -287,6 +287,8 @@ struct _sapi_module_struct {
 	const char *ini_entries;
 	const zend_function_entry *additional_functions;
 	unsigned int (*input_filter_init)(void);
+
+	bool isolate_symbols;
```

Comment:
I would prefer to go for something a bit more generic so we can potentially use it for other things rather than extending the struct everytime.

What about adding `unsigned int flags` field and then `STANDARD_SAPI_MODULE_PROPERTIES_WITH_FLAGS(_flags)` that would allow easily set to any flags in the future. Then you would just define `SAPI_MODULE_FLAG_ISOLATE_SYMBOLS`..?

Reply:
Good idea, done!